quickstep/src/com/android/quickstep/QuickstepProcessInitializer.java

```diff
@@ -64,6 +64,7 @@ protected void init(Context context) {
 
                 // Elevate GPU priority for Quickstep and Remote animations.
                 try {
+                        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.Q) return;
```

Comment:
(self review's note) This could be replace with launcher3's atleast check

https://github.com/LawnchairLauncher/lawnchair/blob/14-dev/src%2Fcom%2Fandroid%2Flauncher3%2FUtilities.java

Reply:
Apply it thanks

Reply:
Applied @ https://github.com/LawnchairLauncher/lawnchair/pull/4331/commits/4190379172ce49d5fda69e704f001c1dd682f232